it/controllers/OverviewPageControllerISpec.scala

```diff
@@ -368,8 +372,6 @@ class OverviewPageControllerISpec extends IntegrationTest with ViewHelpers {
             textOnPageCheck(notStartedText, giftAidStatusSelector)
           }
 
-          textOnPageCheck(specific.goToYourIncomeTax, goToYourIncomeTaxReturnSelector)
-          linkCheck(incomeTaxAccountLink, viewEstimateSelector, Links.viewAndChangeLink(user.isAgent))
```

Comment:
Do we have a test for this content still, somewhere?

Reply:
Updated